snippets/pyaloha/ccode.py

```diff
@@ -151,10 +155,7 @@ def stripped(self):
 def iterate_events(stream_processor, events_limit):
     base_path = os.path.dirname(os.path.abspath(__file__))
     c_module = ctypes.cdll.LoadLibrary(
-        os.path.join(
-            base_path, '..',
-            'c_api', 'build', 'iterate_events.so'
-        )
+        os.path.join(base_path, 'iterate_events.so')
```

Comment:
Please modify Makefile to copy so file to a new location for a legacy usage

Reply:
> Please modify Makefile to copy so file to a new location for a legacy usage

done